### ISSUE TITLE
Dps metre values differ for group members

### ISSUE THREAD

**Issue by Selevan90** [Tercioo/Details-Damage-Meter#818]
Some classes in the war within are showing an obscene dps value in dungeons I’ve seen multiple occurrences of players reaching over 3million dps on level 70 old gear and when questioning all members of the group they are seeing different values for the same player

- Steps to Reproduce (what did you do to make the bug happen):  play a dungeon in the war within with a player not levelled past 70 and wearing the old tier set in some cases classes will show millions in the dps value when some only see 3-400k dps

- Result (what happens when you follow the steps above): dps metre will show over a million on an under leveled character as opposed to level 80 in current expansion world drop gear doing far less damage

- Expected (what you think the expected behavior would be when following the steps): huge dps for low leveled character but lvl 80 character showing minimal dps

- When the bug started (a date, wow path release, details update, etc)? First experienced today 23/08/24 invited a friend to help boost their character and the metre was insane far more then anyone should be at for the current gear and lvl for the expansion

- Additional information:

know classes I’ve seen this on are monk and rogue

**Lua Error**
Post here lua errors, if you aren't getting any, use */console scriptErrors 1*
 No lua errors

**Screenshots**
Post images of the bug if necessary.
![image](https://github.com/user-attachments/assets/a254b00c-cb1d-4fe7-a8fa-61c762b92af2)

**Other Information**
If necessary, add more info here.

**Comment by Flamanis** [comment]
This is how scaling works

**Comment by Selevan90** [comment]
its not scaling when 1 group member sees 3 million damage and another in the same group sees 300k for the same player, there is for certain either an issue with either details or its an issue with the game itself but dps values are not accurate

**Comment by Flamanis** [comment]
Nope, it is entirely how scaling works. Every player sees enemies at their level. For example a level 70 might see the enemy with 1m hp, and the level 80 will see the enemy with 10m hp. If the level 70 does 500k damage total, the level 80 will see the level 70 as having done 5m damage total. In both situations the percent damage done is the same, but the raw numbers are different. This is how scaling works. If the issue persists at max level with all max level players then please report it. But until that time this is intentional from blizzard

**Comment by Flamanis** [comment]
Closing this issue as it is old and could have already been fixed. If you believe this to be in error, or if the issue still persists, please create a new issue.
